scripts/eachdist.py

```diff
@@ -660,7 +649,14 @@ def format_args(args):
     )
     runsubprocess(
         args.dry_run,
-        ("isort", "--settings-path", f"{root_dir}/.isort.cfg", "--profile", "black", "."),
+        (
+            "isort",
```

Comment:
Why do you still keep isort and black here?

Reply:
Oops i missed this thanks

Reply:
Fixed in new PR